### ISSUE TITLE
Tagging files stopped working after update to NC14

### ISSUE THREAD

**Issue by theendcomplete** [nextcloud/server#12707]
### Steps to reproduce
1. Try to tag a file from filelist (with button circled on a screenshot below)
![image](https://user-images.githubusercontent.com/6422361/49152564-5083fe80-f324-11e8-86fe-04f4faadb71a.png) 

### Expected behaviour
After clicking a button, an input field should be shown

### Actual behaviour
nothing happens when you press the button 

### Server configuration

**Operating system**:
Ubuntu Server 16.04.5 LTS 
**Web server:**
nginx/1.10.3 (Ubuntu)

**Database:**
PostgreSQL 11
**PHP version:**
PHP 7.1.24-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Nov 12 2018 09:23:22) ( NTS )
**Nextcloud version:** (see Nextcloud admin page)
14.0.4
**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from nextcloud 13
**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
Enabled:
  - accessibility: 1.0.1
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - dav: 1.6.0
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_pdfviewer: 1.3.2
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
  - gallery: 18.1.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - nextcloud_announcements: 1.3.0
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - password_policy: 1.4.0
  - provisioning_api: 1.4.0
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - support: 1.0.0
  - survey_client: 1.2.0
  - systemtags: 1.4.0
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.1
  - updatenotification: 1.4.1
  - user_ldap: 1.4.0
  - workflowengine: 1.4.0
Disabled:
  - activity
  - admin_audit
  - encryption
  - files_external
  - user_external

</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "14.0.4.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "maintenance": false,
        "loglevel": 2
    }
}

```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
No
**Are you using encryption:** yes/no
only ssl
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
Freeipa 4.4.4
#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
+-------------------------------+--------------------------------------------------------------------------------------------------------+
| Configuration                 |                                                                                                        |
+-------------------------------+--------------------------------------------------------------------------------------------------------+
| hasMemberOfFilterSupport      | 1                                                                                                      |
| hasPagedResultSupport         |                                                                                                        |
| homeFolderNamingRule          | attr:uid                                                                                               |
| lastJpegPhotoLookup           | 0                                                                                                      |
| ldapAgentName                 | uid=cloud-connect,cn=users,cn=accounts,***REMOVED SENSITIVE VALUE***                                            |
| ldapAgentPassword             | ***                                                                                                    |
| ldapAttributesForGroupSearch  |                                                                                                        |
| ldapAttributesForUserSearch   |                                                                                                        |
| ldapBackupHost                |                                                                                                        |
| ldapBackupPort                |                                                                                                        |
| ldapBase                      | ***REMOVED SENSITIVE VALUE***                                                                                  |
| ldapBaseGroups                | cn=groups,cn=accounts,***REMOVED SENSITIVE VALUE***                                                             |
| ldapBaseUsers                 | cn=users,cn=accounts,***REMOVED SENSITIVE VALUE***                                                              |
| ldapCacheTTL                  | 600                                                                                                    |
| ldapConfigurationActive       | 1                                                                                                      |
| ldapDefaultPPolicyDN          |                                                                                                        |
| ldapDynamicGroupMemberURL     |                                                                                                        |
| ldapEmailAttribute            | mail                                                                                                   |
| ldapExperiencedAdmin          | 0                                                                                                      |
| ldapExpertUUIDGroupAttr       |                                                                                                        |
| ldapExpertUUIDUserAttr        | mail                                                                                                   |
| ldapExpertUsernameAttr        | mail                                                                                                   |
| ldapGidNumber                 | gidNumber                                                                                              |
| ldapGroupDisplayName          | cn                                                                                                     |
| ldapGroupFilter               | (&(|(objectclass=posixgroup))(|(memberOf=cn=mailing_list,cn=groups,cn=accounts,***REMOVED SENSITIVE VALUE***))) |
| ldapGroupFilterGroups         |                                                                                                        |
| ldapGroupFilterMode           | 1                                                                                                      |
| ldapGroupFilterObjectclass    |                                                                                                        |
| ldapGroupMemberAssocAttr      | member                                                                                                 |
| ldapHost                      | ***REMOVED SENSITIVE VALUE***                                                                             |
| ldapIgnoreNamingRules         |                                                                                                        |
| ldapLoginFilter               | (&(|(objectclass=inetorgperson))(|(uid=%uid)(mail=%uid)))                                              |
| ldapLoginFilterAttributes     | cn;memberOf                                                                                            |
| ldapLoginFilterEmail          | 0                                                                                                      |
| ldapLoginFilterMode           | 1                                                                                                      |
| ldapLoginFilterUsername       | 1                                                                                                      |
| ldapNestedGroups              | 1                                                                                                      |
| ldapOverrideMainServer        |                                                                                                        |
| ldapPagingSize                | 500                                                                                                    |
| ldapPort                      | 389                                                                                                    |
| ldapQuotaAttribute            | employeeNumber                                                                                         |
| ldapQuotaDefault              | 1048576                                                                                                |
| ldapTLS                       | 0                                                                                                      |
| ldapUserAvatarRule            | default                                                                                                |
| ldapUserDisplayName           | mail                                                                                                   |
| ldapUserDisplayName2          | displayname                                                                                            |
| ldapUserFilter                | (|(objectclass=inetOrgPerson))                                                                         |
| ldapUserFilterGroups          | ipausers                                                                                               |
| ldapUserFilterMode            | 1                                                                                                      |
| ldapUserFilterObjectclass     | organizationalperson;posixAccount                                                                      |
| ldapUuidGroupAttribute        | auto                                                                                                   |
| ldapUuidUserAttribute         | auto                                                                                                   |
| turnOffCertCheck              | 0                                                                                                      |
| turnOnPasswordChange          | 0                                                                                                      |
| useMemberOfToDetectMembership | 1                                                                                                      |
+-------------------------------+--------------------------------------------------------------------------------------------------------+

```
</details>

### Client configuration
**Browser:**
Chrome  70.0.3538.77
Firefox 63.0.3 (64 bit

**Operating system:**
Linux Mint Tara

#### Web server error log
<details>
<summary>Web server error log</summary>

```
no visible errors about this
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
no visible errors about this

```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

the only error in Console log:

Error in event handler for (unknown): TypeError: Cannot read property 'childElementCount' of null
    at Object.getImages (chrome-extension://immpkjjlgappgfkkfieppnmlhakdmaab/includes/content.js:32:340)
    at Object.find (chrome-extension://immpkjjlgappgfkkfieppnmlhakdmaab/includes/content.js:47:330)
    at Object.m_over (chrome-extension://immpkjjlgappgfkkfieppnmlhakdmaab/includes/content.js:113:10)
    at Object.PVI.initOnMouseMoveEnd (chrome-extension://immpkjjlgappgfkkfieppnmlhakdmaab/includes/content.js:140:84)
    at init (chrome-extension://immpkjjlgappgfkkfieppnmlhakdmaab/includes/content.js:136:508)

```
</details>

**Comment by sophocles1942** [comment]
We are having the same issue where the tag icon is no longer working.

### Steps to reproduce
1. Same as above.

### Expected behaviour
Same as above

### Actual behaviour
Same as above

### Server configuration

**Operating system**: CentOS V6.10

**Web server:** Apache/Nginx

**Database:** 10.2.19-MariaDB

**PHP version:** PHP 7.2 FPM

**Nextcloud version:** 14.0.4

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from nextcloud 14 and before that 13

**Where did you install Nextcloud from:**
Official download link on website

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

### Logs
#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by ChristophWurst** [comment]
Confirmed. Doesn't work on 14.

**Comment by Spartachetto** [comment]
Dumb question: is this different from #12810 ?

**Comment by MorrisJobke** [comment]
Fixed in 14.0.5 - see https://github.com/nextcloud/server/issues/12810#issuecomment-447630594
